engine/api/services/gorp_model.go

```diff
@@ -12,7 +12,9 @@ type service struct {
 }
 
 func (s service) Canonical() gorpmapper.CanonicalForms {
+	_ = []interface{}{s.ID, s.Name, s.Type, s.Region, s.IgnoreJobWithNoRegion} // Checks that fields exists at compilation
 	return []gorpmapper.CanonicalForm{
+		//"{{.ID}}{{.Name}}{{.Type}}{{.Region}}{{.IgnoreJobWithNoRegion}}",
```

Comment:
I think we can remove the comment for this line

Reply:
Will do in a next PR.